### PR TITLE
[#54] List versions

### LINKED ISSUE #54: List current index.html versions
ember deploy:versions

### PR DESCRIPTION
**Pull request by pavloo**
Closes #54

### CODE REVIEW

**Comment by pavloo** [comment]
@achambers any expectations on when it's gonna be merged/reviewed ?

**Comment by achambers** [comment]
Sorry mate. I'm on my way to Australia (from London) and got stuck in Thailand for a few days longer than expected due to my baby ending up in hospital. Hence I haven't had a chance to think about it. Will be in Oz in a couple of days where I can sit down and review/merge for you. So by the end if the week it'll be done. 

Sorry for the delay. Thanks for your work.

**Comment by pavloo** [comment]
@achambers Oh, I hope everything will be ok with you and your family. Sorry.

**Comment by achambers** [comment]
It's all good now mate. Just a slight hiccup in our plans ;) Will get this in as soon as I can.

File: tests/unit/commands/deploy-index-test.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var Promise      = require('ember-cli/lib/ext/promise');
 var MockRegistry = require('../../helpers/mock-registry');
 
 describe('deploy:index command', function() {
-  var subject
+  var subject;
```
Comment (achambers) [inline tests/unit/commands/deploy-index-test.js]: Please remove the whitespace from the end of this line.

File: tests/unit/commands/deploy-versions-test.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var assert       = require('ember-cli/tests/helpers/assert');
+var Task         = require('ember-cli/lib/models/task');
+var Promise      = require('ember-cli/lib/ext/promise');
+var MockRegistry = require('../../helpers/mock-registry');
+
+describe('deploy:versions command', function() {
+  var subject,
+      tasks;
```
Comment (achambers) [inline tests/unit/commands/deploy-versions-test.js]: Please add `var` to this line.

File: tests/unit/tasks/deploy-versions-test.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+var assert      = require('ember-cli/tests/helpers/assert');
```
Comment (achambers) [inline tests/unit/tasks/deploy-versions-test.js]: Could you please just indent these lines so that they line up?

**Comment by achambers** [comment]
Mate, thanks for this.  Just having a look at it.  There were just a couple of very minor things I'd like to change if that's cool.  I'd change them myself but not sure how to myself seeing it's on your branch.  So if you could do that and update the PR, I'll happily merge in.  Have a great xmas mate.

**Comment by achambers** [comment]
Also, you will need to rebase against the `next-release-v0.0.6` branch.  And if you could squish the commits, that'd be great.

**Comment by pavloo** [comment]
@achambers fixed your comments, rebased and squashed - looks like ready to be merged. Marry Christmas !

**Comment by achambers** [comment]
Thanks so much for that mate.  Great work.  Have a great Christmas yourself.

**Pull request merged**